doc/ref_model/chapters/chapter10.md

```diff
@@ -40,15 +40,20 @@ Gaps
 
 This section addresses major open issues identified in the development of the Reference Model, Reference Architecture, Reference Implementation of the Common Cloud Infrastructure Lifecycle Framework. 
 
-## 10.3.1 Discovery [Initial language]
-The VNFs and NFVI frameworks should be able to discover each other and exchange their capabilities required or offered. CNTT reference model specifies a model, but OpenStack does not provide APIs to support such capabilities. Following should be supported: NFVI should be able to publish the capabilities it offers to VNFs/CNFs VNFs/CNFs should be able to query the NFVI for specific capabilities - such as number of cores, performance parameters. Negotiation/Hand Shake API - VNFs and NFVIs should be able to negotiate on certain capabilities. For instance, VNF desires HW acceleration for high throughput, but, should be able to fall back to high throughput offered by NFVI via DPDK offering, and vice-a-versa.
-
-## 10.3.2 Support Load Balance of VNF/CNFs [Initial language]
+## 10.3.1 Discovery
+The workloads (VNFs/CNFs) and Cloud Infrastructure should be able to discover each other and exchange their capabilities required or offered. One of the key pain points for most of the operators is the VNF/CNF onboarding - both in terms of time and complexity. It could take weeks or months to onboard a VNF/CNF. There are lots of static and laborious checks performed to ensure the compatibility of the workloads with the corresponding Cloud Infrastructure. 
```

Comment:
I would avoid that statement "It could take weeks or months to onboard a VNF/CNF. "
It doesn't refer to CPU time or execution time on the infrastructure, but manual preparation before the onboarding can start.
Also the extensive checks to ensure compatibility or workloads and infrastructure are not a disadvantage. But the need to use them by try and error is a problem. It should be solved by automation, which is said in the next sentence anyway.
Suggest:
```suggestion
The workloads (VNFs/CNFs) and Cloud Infrastructure should be able to discover each other and exchange their capabilities required or offered. One of the key pain points for most of the operators is the required preparation effort for VNF/CNF onboarding - both in terms of time and complexity.  
```